### PR TITLE
[ISSUE-647] e2e test artifacts

### LINKED ISSUE #647: Attach new e2e test logs to GitHub action artifacts
Make artifacts available in GH workflow

### PR DESCRIPTION
**Pull request by rkiselenko**
### Resolves #647

Make artifacts available in GH workflow
Fix on comment trigger git ref

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Provide test details_

### CODE REVIEW

**Review by misanthropicat — APPROVED** [review]

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 name: e2e
 
 on:
+  pull_request:
```
Comment (safronovD) [inline .github/workflows/e2e.yml:4]: Why do we need this trigger? e2e tests should be optional to not spend Github runners time

Comment (rkiselenko) [reply .github/workflows/e2e.yml:4]: Removed

**Review by safronovD — APPROVED** [review]

**Pull request merged**